apps/labs/posts/sangho-internship-blogpost.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Sangho's Internship at Quansight"
```

Comment:
I suggest picking a title which is more related to your project's work, not everyone who reads Quansight Labs' blog will know who you are after all 😉. If I scroll through our Blog pages, how would I know what your internship/blog is about?